packages/flutter_driver/lib/src/extension/extension.dart

```diff
@@ -229,6 +232,10 @@ class FlutterDriverExtension {
 
   Future<Health> _getHealth(Command command) async => const Health(HealthStatus.ok);
 
+  Future<LayerTree> _getLayerTree(Command command) async {
+    return LayerTree(RendererBinding.instance?.renderView?.debugLayer?.toStringDeep());
```

Comment:
This will not work in profile/release mode - is that ok?

Reply:
(or rather, it will return `null` in those cases)

Reply:
I think it's fine, this should match the existing behaviour (https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/rendering/binding.dart#L448)